<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>lister</class>
 <widget class="QDialog" name="lister">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>721</width>
    <height>553</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>581</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	font-size: 25px;
}</string>
   </property>
   <property name="text">
    <string>séléctionner tous les etudiants de meme :</string>
   </property>
  </widget>
  <widget class="QPushButton" name="quiter_butt">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>280</y>
     <width>101</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Quitter</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radio_nom">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>180</y>
     <width>241</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Nom</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>330</y>
     <width>281</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	font-size: 25px;
}</string>
   </property>
   <property name="text">
    <string>Affichage :</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radio_valid">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>240</y>
     <width>161</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Validée un module</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radio_ens">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>210</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Validée la semestre</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radio_prenom">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>150</y>
     <width>241</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Prénom</string>
   </property>
  </widget>
  <widget class="QPushButton" name="select_Button">
   <property name="geometry">
    <rect>
     <x>109</x>
     <y>280</y>
     <width>101</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Sélectionner</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>-40</x>
     <y>310</y>
     <width>741</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>-10</y>
     <width>631</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
font-size:40px;
	}</string>
   </property>
   <property name="text">
    <string>Lister Les Etudiants :</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="text_result">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>330</y>
     <width>371</width>
     <height>211</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="module_inp">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>240</y>
     <width>201</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
